tracee-ebpf/tracee/net_events.go

```diff
@@ -30,83 +38,39 @@ func (t *Tracee) processNetEvents() {
 
 			evtMeta, dataBuff := parseEventMetaData(in)
 
-			// timeStamp is nanoseconds since system boot time
 			timeStampObj := time.Unix(0, int64(evtMeta.TimeStamp+t.bootTime))
 
-			if evtMeta.NetEventId == NetPacket {
-				var pktLen uint32
-				err := binary.Read(dataBuff, binary.LittleEndian, &pktLen)
-				if err != nil {
+			captureData, err := parseCaptureData(dataBuff)
```

Comment:
Capture data is not available when only `t.config.Debug` is set and not `--capure net` - reading it here will break this flow